docker-compose.yml

```
@@ -21,7 +21,8 @@ services:
       - qdrant_data:/qdrant/storage
     restart: always
     environment:
-      - QDRANT_HOST=0.0.0.0
+        QDRANT_DB_HOST: 0.0.0.0
+        QDRANT_DB_PORT: 6333
```

This seems to be wrong

https://qdrant.tech/documentation/guides/configuration/#environment-variables
https://github.com/qdrant/qdrant/blob/master/config/config.yaml

May we need this instead?
```suggestion
        QDRANT__SERVICE__HOST: 0.0.0.0
        QDRANT__SERVICE__HTTP_PORT: 6333
```

---

Let's add above links as NOTE before theses

---

done